services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/standards_standards_progress_report.jsx

```diff
@@ -47,43 +49,50 @@ export default class StandardsProgressReport extends React.Component {
         accessor: 'standard_level',
         sortType: sortTableByStandardLevel,
         resizable: false,
-        width: 150
+        maxWidth: 150
       }, {
         Header: "Standard name",
         accessor: 'standard_name',
         sortType: sortTableByStandardLevel,
-        minWidth: 200,
+        minWidth: 500,
         resizable: false,
-        Cell: ({row}) => (
-          <a className='row-link-disguise underlined' href={`/teachers/progress_reports/standards/classrooms/0/standards/${row.original['id']}/students`}>
-            {row.original['standard_name']}
-          </a>
-        )
+        Cell: ({ row }) => {
+          const { original } = row
+          const { id, name, standard_students_href } = original
+          return renderTooltipRow({ id, label: name, link: standard_students_href, headerWidth: 500 })
+        },
       }, {
         Header: 'Activities',
         accessor: 'total_activity_count',
-        width: 115,
+        maxWidth: 100,
         resizable: false
       }, {
         Header: 'Time spent',
         accessor: 'timespent',
         resizable: false,
-        width: 100,
+        maxWidth: 100,
         Cell: ({ row }) => <span className={blurIfNotPremium}>{getTimeSpent(row.original['timespent'])}</span>
       }, {
-        Header: 'Avg. score',
+        Header: 'Average score',
         accessor: 'average_score',
         resizable: false,
-        width: 100,
+        maxWidth: 160,
         Cell: ({ row }) => <span className={blurIfNotPremium}>{`${row.original['average_score']}`}</span>
       }, {
         Header: 'Proficiency Status',
         accessor: 'mastery_status',
         resizable: false,
-        width: 165,
-        Cell: ({row}) => (
-          <span className={blurIfNotPremium}><span className={row.original['mastery_status'] === 'Proficient' ? 'proficient-indicator' : 'not-proficient-indicator'} />{row.original['mastery_status']}</span>
-        )
+        maxWidth: 200,
+        Cell: ({ row }) => {
+          const proficiencyStatus = row.original['mastery_status'] === 'Proficient' ? PROFICIENT : 'not-proficient'
+          const iconSrc = `${process.env.CDN_URL}/images/icons/2xs/${proficiencyStatus === PROFICIENT ? 'proficiency-circle/proficient' : 'proficiency-circle/no-proficiency'}.svg`
```

Comment:
It feels a little weird to me to be dynamically building the URL for this icon based on a variable.  Does it make sense to create two constants, one for each URL, and then pick one based on the condition instead?  That would at least make it possible to search the codebase for references to a file and find them, whereas this code, if I searched for `images/icons/2xs/proficiency-circle/proficient.svg` I wouldn't find this code.

Reply:
ok, that's a fair point-- I'll update